win/build-deps.cmd

```diff
@@ -115,8 +119,13 @@ call cecho.cmd 0 14 "Warning: You will need roughly 8 GB of disk space to procee
 echo.
 
 call cecho.cmd black cyan "If you are not ready with the above, press Ctrl-C to abort!"
```

Comment:
@Stinkfist0 Sometimes you use `utils\cecho.cmd`, sometimes just `cecho.cmd`. I never observed any problems with this, but now I'm running this on Windows 10 (for my build farm) and I get many complaints about paths not found. Same for `7za`. The only way I can get this to work reliably is to use `%~dp0\utils\cecho.cmd` and `%~dp0\utils\7za`. Did you test on Windows 10? Isn't it weird this actually works with all the `pushd`s and `popd`s flying around? Does it makes sense to change this to use full paths?